CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 [//]: # (comment: Don't forget to update statsd/telemetry.go:clientVersionTelemetryTag when releasing a new version)
 
+# 5.2.0 / 2022-12-21
+
+- [BETA][FEATURE] Support sending counters and gauges with timestamp. See [#262][]
```

Comment:
Don't forget to add the reference to 262 & 269 at the very of the document, same goes with byronwolfman handle.